DuckDuckGo/OnboardingExperiment/Pixels/OnboardingPixelReporter.swift

```diff
@@ -174,15 +174,15 @@ extension OnboardingPixelReporter: OnboardingIntroPixelReporting {
 extension OnboardingPixelReporter: OnboardingSearchSuggestionsPixelReporting {
```

Comment:
The ViewModel is defined in BSK as it's shared between iOS and macOS. macOS will send these pixels during the experiment.

I can create a different init using #if os(iOS) but since these pixels are temporary on macOS, I thought we can just make a BSK PR once the experiment finishes in macOS.